src/KOKKOS/fft3d_kokkos.cpp

```diff
@@ -684,6 +688,16 @@ struct fft_plan_3d_kokkos<DeviceType>* FFT3dKokkos<DeviceType>::fft_3d_create_pl
                        NULL,&nslow,1,plan->length3,
                        NULL,&nslow,1,plan->length3,
                        FFTW_BACKWARD,FFTW_ESTIMATE);
+
```

Comment:
This looks like duplicated code, can the 3 instances be moved below to a single instance?

Reply:
I mentioned this in my e-mail to you. I think this had to be different for several different FFTs before we reduced the options to FFTW, MKL, and KISS. It was incorrectly applied twice for the MKL version in `fft3d_kokkos.cpp` so I made it consistent with `fft3d.cpp`. I have no problem with removing the redundancy, but would then want to do it consistently across both 3d-FFT wrapper files.

Reply:
OK I support removing the redundancy.

Reply:
done